department-of-veterans-affairs/caseflow
# Restore Removed Hearing Day Confirmation Message

Resolves #8825 

### Description
Fixed a prop name from hearingDate to scheduledFor so that the remove hearing day confirmation shows up again in the Hearing Schedule welcome page.

### Acceptance Criteria

### Testing Plan
1. Remove a Hearing Day
2. Confirm the confirmation banner shows up in the Welcome To Hearing Schedule page with the correct date that was removed.
